apps/condo/domains/common/components/EmptyListView.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled'
+import { Empty, Space, Typography } from 'antd'
+import { useRouter } from 'next/router'
+import { useIntl } from '@core/next/intl'
+import { Button } from './Button'
+import { EmptyIcon } from './EmptyIcon'
+
+const EmptyListViewContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+`
+export const EmptyListView = () => {
+    const intl = useIntl()
+    const router = useRouter()
+
+    return (
+        <EmptyListViewContainer>
```

Comment:
Why we don't use or customize the https://ant.design/components/empty/

Reply:
Maybe it's better to use `renderEmpty` of https://ant.design/components/config-provider/